src/internal/proc.js

```diff
@@ -403,6 +403,13 @@ export default function proc(
     const cancelPromise = promise[CANCEL]
     if(typeof cancelPromise === 'function') {
       cb.cancel = cancelPromise
+    } else {
```

Comment:
hm, could u be more precise? i dont see any local variable here named `abort`, the only one is `promise`